### PR TITLE
Stringify qualified keywords with namespaces

### PR DESCRIPTION
**Pull request by just-sultanov**
Signed-off-by: Ilshat Sultanov <just.sultanov@gmail.com>

### CODE REVIEW

**Review by federkasten — APPROVED** [review]
LGTM 👍  Just a comment.

File: src/clucie/core.clj [hunk 1]
```diff
@@ -13,7 +13,9 @@
   (cond
     (string? v) v
     (integer? v) (str v)
-    (keyword? v) (name v)
+    (keyword? v) (if (some? (namespace v))
+                   (-> v str (subs 1))
```
Comment (federkasten) [inline src/clucie/core.clj:17]: It may be better to commonize this stringify function.

**Review by totakke — APPROVED** [review]
LGTM

File: src/clucie/core.clj (hunk 1 above)
Comment (just-sultanov) [reply src/clucie/core.clj:17]: Yes, it's a good idea

**Pull request merged**